inst/examples/dashboard/header.R

```diff
@@ -41,23 +41,112 @@ farItems <- list(
     ariaLabel = "Grid view",
     iconOnly = TRUE,
     iconProps = list(iconName = "Tiles")
-  ),
-  list(
-    key = "info",
-    id = "aboutInfo",
-    href = "#!/about",
-    text = "Info",
-    ariaLabel = "Info",
-    iconOnly = TRUE,
-    iconProps = list(iconName = "Info")
   )
 )
 
-command_bar <- CommandBar(items = items,
+command_bar <- CommandBar(
+  items = items,
   farItems = farItems,
-  style = list(width = "100%"))
+  style = list(width = "100%")
+)
+
+
+appsilon_header <- function() {
+
+  appsilon_breakpoints <- breakpointSystem(
+    "appsilon-breakpoints",
+    breakpoint("xs", min = 320),
+    breakpoint("s", min = 428),
+    breakpoint("m", min = 728),
+    breakpoint("l", min = 1024),
+    breakpoint("xl", min = 1200)
+  )
+
+  gridPanel(
+    id = "app_header",
+
+    class = "mobile-collapsed",
+
+    breakpoint_system = appsilon_breakpoints,
+
+    areas = list(
```

Comment:
Use blank lines sparingly. See [this](https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace) for rationale. Ditto for the rest of the file.

```suggestion
    id = "app_header",
    class = "mobile-collapsed",
    breakpoint_system = appsilon_breakpoints,
    areas = list(
```